README.md

```diff
@@ -9,11 +9,14 @@ Cards for Humanity is a fast-paced online version of the popular card game, Card
 
 Our Team
 --------
-Cards for Humanity was created at [Hack Reactor](http://www.hackreactor.com) by:
-* [Matt Silverstein](http://www.mattsilverstein.com/)
-* [Will Ngo](https://mrngoitall.net)
-* [Tyler McGinnis](http://www.tylermcginnis.com)
-* [David Gonzalez](http://www.truthyfalsy.com)
+Cards for Humanity was created at [Andela](https://www.andela.com) by:
```

Comment:
I don't agree with the wording you used here, this game wasn't created by us.  Upgraded maybe? but definitely not created